### ISSUE TITLE
Can we send image with captions?

### ISSUE THREAD

**Issue by nirajmohanrana** [devwithkunal/whatsapp-bulk-messenger#1]

**Comment by devwithkunal** [comment]
Yes, you can, currently, it's on the Experimental Branch, set the configs and run the `main_2.py` file from the Experimental Branch. Make sure to set the image path as an absolute path.
